requirements.txt

```diff
@@ -0,0 +1,29 @@
+boto==2.40.0
+colander==1.3.1
+colorama==0.3.7
+cornice==1.2.1
+enum34==1.1.6
+functools32==3.2.3.post2
+iso8601==0.1.11
+jsonschema==2.5.1
+kinto==3.1.1
+PasteDeploy==1.5.2
+pyramid==1.7
```

Comment:
The requirements.txt file is just there as the known good version set it is not used to actually install the pluging.